lib/swiss_db/db.rb

```diff
@@ -30,7 +30,8 @@ def schema(schema_name, &block)
     puts @current_schema.inspect
   end
 
-  def entity(table_name, &block)
+  def entity(class_name, &block)
+    table_name = class_name.tableize
```

Comment:
This is the only instance of class_name and table_name being backwards that I could find. Please let me know if you see any others.